frontend/www/js/omegaup/components/problem/CollectionList.vue

```diff
@@ -4,14 +4,31 @@
     <div class="row">
       <div class="col col-md-4">
         <omegaup-problem-filter-tags
-          :tags.sync="tags"
+          :current-tags="currentTags"
+          :tags.sync="availableTags"
```

Comment:
`availableTags` es un getter, entonces no puede ir con `.sync`: si no, al momento de que el componente hijo intente actualizarlo, va a arrojar una excepción!

```suggestion
          :tags="availableTags"
```